compiler/cpp/src/lexer/Token.h

```diff
@@ -10,7 +10,7 @@
 
 class Token {
 private:
-    int type = 0;
+    int type = -1;
```

Comment:
That's a C++ kind of thing that you have to init the variable here already, right?

Reply:
Yes, unfortulately ...

Reply:
;(